diff -urN qutim-0.2.1.git.630/src/corelayers/contactlist/contactlistproxymodel.cpp qutim-0.2.1.git.630-patched/src/corelayers/contactlist/contactlistproxymodel.cpp
--- qutim-0.2.1.git.630/src/corelayers/contactlist/contactlistproxymodel.cpp	2011-09-15 00:57:33.000000000 +1100
+++ qutim-0.2.1.git.630-patched/src/corelayers/contactlist/contactlistproxymodel.cpp	2012-11-02 00:01:54.000000000 +1100
@@ -98,6 +98,11 @@
 {
 	return 1;
 }
+bool ContactListProxyModel::hasChildren(const QModelIndex &parent) const
+{
+	return QAbstractItemModel::hasChildren(parent);
+}
+
 QVariant ContactListProxyModel::data(const QModelIndex &index, int role) const
 {
 	if(index.column()!=0)
diff -urN qutim-0.2.1.git.630/src/corelayers/contactlist/contactlistproxymodel.h qutim-0.2.1.git.630-patched/src/corelayers/contactlist/contactlistproxymodel.h
--- qutim-0.2.1.git.630/src/corelayers/contactlist/contactlistproxymodel.h	2011-09-15 00:57:33.000000000 +1100
+++ qutim-0.2.1.git.630-patched/src/corelayers/contactlist/contactlistproxymodel.h	2012-11-02 00:01:06.000000000 +1100
@@ -44,6 +44,7 @@
 
 	int rowCount(const QModelIndex &parent = QModelIndex()) const;
 	int columnCount(const QModelIndex &parent = QModelIndex()) const;
+	virtual bool hasChildren(const QModelIndex &parent = QModelIndex()) const;
 
 	Qt::ItemFlags flags(const QModelIndex &index) const;
 	Qt::DropActions supportedDropActions() const;
